src/components/TodoError.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from 'react';
+import classNames from 'classnames';
+import { Error } from '../types/Error';
+
+type Props = {
+  error: Error,
+  onErrorChange: (error: Error) => void,
+};
+
+export const TodoError: React.FC<Props> = ({ error, onErrorChange }) => {
+  useEffect(() => {
+    if (error) {
+      const timeotId = setTimeout(() => onErrorChange(Error.None),
+        3000);
+
+      clearTimeout(timeotId);
```

Comment:
The clearTimeout function should also be called in the cleanup function of useEffect to clear the timeout when the component unmounts or when the error changes.